bin/email-failed-echo-bulky-payment

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+
+BEGIN {
+    use File::Basename qw(dirname);
+    use File::Spec;
+    my $d = dirname( File::Spec->rel2abs($0) );
+    require "$d/../setenv.pl";
+}
+
+use DateTime;
+use FixMyStreet::Cobrand;
+use FixMyStreet::DB;
+use FixMyStreet::Email;
+use mySociety::Locale;
+
+mySociety::Locale::gettext_domain('FixMyStreet');
+
+my $comment_text = 'Echo failure email sent';
+
+my %cobrands = (
+    kingston => FixMyStreet::Cobrand->get_class_for_moniker('kingston')->new,
+    sutton   => FixMyStreet::Cobrand->get_class_for_moniker('sutton')->new,
+);
+
+my $dtf         = FixMyStreet::DB->schema->storage->datetime_parser;
+my $cutoff_date = $dtf->format_datetime(
+    DateTime->now( time_zone => 'Europe/London' )->subtract( hours => 1 ) );
+
+my $rs = FixMyStreet::DB->resultset('Problem')->search(
+    {   category => 'Bulky collection',
+        cobrand  => [qw/kingston sutton/],
+        created  => { '<'  => $cutoff_date },
+        extra    => { '\?' => 'payment_reference' },
+    },
+);
+
+while ( my $report = $rs->next ) {
+    # Ignore if there is an update in Comment table with an external_id, or
+    # failure email has been sent previously
+    my $rs_comment = FixMyStreet::DB->resultset('Comment')->search(
+        {   problem_id => $report->id,
+            -or => [ text => $comment_text, external_id => { '!=', undef } ],
+        }
+    );
+
+    next if $rs_comment->count > 0;
+
+    # Send email
+    my $email
+        = $report->cobrand eq 'kingston'
+        ? 'uk.kingston.collections@veolia.com'
+        : 'neighbourhoodservices@sutton.gov.uk';
+
+    my $cobrand = $cobrands{ $report->cobrand };
+
+    my $h = {
+        report  => $report,
+        cobrand => $cobrand,
+    };
+
+    FixMyStreet::Email::send_cron(
+        FixMyStreet::DB->schema,
+        'waste/echo_failed_bulky_payment.txt',
+        $h,
+        {   To   => $email,
+            From => [
+                FixMyStreet->config('CONTACT_EMAIL'),
+                FixMyStreet->config('CONTACT_NAME'),
+            ],
+        },
+        undef,    # env_from
+        undef,    # nomail
+        $cobrand,
+        $report->lang,
+    );
+
+    my $comment = FixMyStreet::DB->resultset('Comment')->new(
+        {   problem => $report,
+            text    => $comment_text,
```

Comment:
I don't think we want to leave a public update on a report (that would get sent to the user). Instead of this, I would set a flag on the report object metadata, as some other scripts do (e.g. send-claims-to-echo sets `sent_to_evo` to 1).